plugins/woocommerce-blocks/assets/js/base/components/cart-checkout/totals/coupon/index.tsx

```diff
@@ -46,28 +46,16 @@ export const TotalsCoupon = ( {
 	displayCouponForm = false,
 }: TotalsCouponProps ): JSX.Element => {
 	const [ couponValue, setCouponValue ] = useState( '' );
-	const [ isCouponFormHidden, setIsCouponFormHidden ] = useState(
-		! displayCouponForm
-	);
+	const [ isCouponFormVisible, setIsCouponFormVisible ] =
```

Comment:
Changed this to a visible instead of hidden to align with the state in the Panel component and have less negation everywhere

Reply:
That's a great improvement! 🙌